### DISCUSSION TITLE
Is it possible to attach a reverb to this player?

### DISCUSSION THREAD

**Discussion by borisorlov21** [mpv-player/mpv#14684] [Q&A] [answered]
### Expected behavior of the wanted feature

Is it possible to attach a reverb to this player?

**Comment by CounterPillow** [comment, marked as answer]
You can load LADSPA audio plugins through [`--af=ladspa=pluginhere`](https://ffmpeg.org/ffmpeg-filters.html#toc-ladspa) or apply arbitrary audio filtering by running mpv's audio output through a plugin host.

Please do not use the issue tracker for asking questions, that's what [Discussions](https://github.com/mpv-player/mpv/discussions) is for.
